pkg/webhook/pod/util.go

```diff
@@ -69,7 +68,7 @@ func addDeferDeletingToPVC(podAC *PodAdmissionControl, tc *v1alpha1.TidbCluster,
 
 // check whether the former upgraded pd pods were healthy in PD cluster during PD upgrading.
 // If not,then return an error
-func checkFormerPDPodStatus(kubeCli kubernetes.Interface, pdClient pdapi.PDClient, tc *v1alpha1.TidbCluster, namespace string, ordinal int32, replicas int32) error {
+func checkFormerPDPodStatus(podLister corelisters.PodLister, pdClient pdapi.PDClient, tc *v1alpha1.TidbCluster, namespace string, ordinal int32, replicas int32) error {
```

Comment:
Not sure if it's reasonable, just feel a little wired to have a lister as a function argument.

Reply:
It only used in pd_deleter, so I think that's ok.